common/game_rules/000_wc_game_rules.txt

```diff
@@ -3,4 +3,13 @@
 
 	wc_lore_characters_generate_families_on = {  }
 	wc_lore_characters_generate_families_off = {  }
+}
+
+wc_interracial_marriage = {
```

Comment:
I feel like we need 2 game rules:
One is responsible for the negative marriage opinion.
Another is responsible for the marriage possibility.
Thus you can have all marriages possible, but with a negative opinion.

Reply:
Added in d3b6192